### PR TITLE
fix: Cypress post upgrade

### PR DESCRIPTION
**Pull request by JoakimSM**
Changes:
- Automatically start the app when running `yarn cy:open` and `yarn cy:run`
- Some time ago we made some custom scripts in our app to deal with some Cypress edge cases. An example would be be if you create a `.env.development` or `.env.development.local` file locally and then run `yarn cy:open`; in this case you would, without the scripts, run against your local instance while the api requests made in your tests will run against your test instance. To fix this we specify our Cypress config in the `.env.cypress` file and the server specified here will always be used. Updated the variable names in this file instead of `cypress.json`.
- Fixed some tests after updating the DB

### CODE REVIEW

File: cypress/plugins/index.js [hunk 1]
```diff
@@ -8,7 +7,6 @@ const getCypressEnvVariables = require('./getCypressEnvVariables');
 const path = require('path');
 
 module.exports = (on, config) => {
-    networkShim(on);
```
Comment (JoakimSM) [inline cypress/plugins/index.js:11]: @simonadomnisoru I don't think we need this, but I might be wrong. Any insights?

File: cypress/support/index.js [hunk 2]
```diff
@@ -12,4 +10,3 @@ beforeEach(() => {
     */
     cy.clearCookies({ domain: null });
 });
-enableNetworkShim();
```
Comment (JoakimSM) [inline cypress/support/index.js:15]: @simonadomnisoru same as other comment.

File: package.json [hunk 3]
```diff
@@ -72,8 +72,8 @@
     "flow:check": "./node_modules/.bin/flow check ./src",
     "flow:addTypes": "flow-typed install",
     "lint": "npm run flow:check && npm run linter:check",
-    "cy:open": "wait-on 'http-get://localhost:3000' && cypress open",
-    "cy:run": "wait-on 'http-get://localhost:3000' && cypress run",
+    "cy:open": "concurrently --kill-others \"yarn start:forCypress\" \"wait-on 'http-get://localhost:3000' && cypress open\"",
```
Comment (simonadomnisoru) [inline package.json:75]: Good idea!

File: cypress/plugins/index.js (hunk 1 above)
Comment (simonadomnisoru) [reply cypress/plugins/index.js:11]: You're right. We don't need it. We use live network requests therefore there is no need to record them and play them back.

**Review by simonadomnisoru — APPROVED** [review]

**Pull request merged**